GVRf/Extensions/gvrf-physics/src/main/java/org/gearvrf/physics/GVRWorld.java

```diff
@@ -113,9 +117,15 @@ public void removeBody(GVRRigidBody gvrBody) {
 
     @Override
     public void onDrawFrame(float frameTime) {
-        NativePhysics3DWorld.step(getNative(), frameTime);
-
+        mFrameTime += frameTime;
+        if (mIsProcessing) {
+            return;
+        }
+        mIsProcessing = true;
+        NativePhysics3DWorld.step(getNative(), mFrameTime);
```

Comment:
Is this getting called more than once per frame somehow? How is step exactly getting re-entered?

When I removed these changes but kept the mesh nullptr check there were no crashes.